integration_test/support/schemas.exs

```diff
@@ -26,7 +26,7 @@ defmodule Ecto.Integration.Post do
   use Ecto.Integration.Schema
   import Ecto.Changeset
 
-  schema "posts" do
+  schema Ecto.Integration, "posts" do
```

Comment:
I'm in doubt on whether I should replicate the assocs integration test (integration_test/cases/assoc.exs) & the associated schemas (integration_test/support/schemas.exs) to use a scoped schema in the tests.

I scoped the schemas here because it was easier, but I would like to hear other opinions on this.